FontAwesome.Sharp/WinForms/FormsIconHelper.cs

```diff
@@ -118,6 +118,7 @@ public static void AddIcons(this ImageList imageList, int size, Color color, par
 
         internal static FontFamily FontFamilyFor(IconChar iconChar)
         {
+            if (Fonts == null) throw new Exception("FontAwesome source font files not found!");
```

Comment:
This is the fix? Ok, let's merge it in then. I would probably change it to a more specific exception, e.g. `InvalidOperationException`

Reply:
`if (Fonts ==` Just accessing the field causes the initializer to be called, making sure that the font data is loaded into memory before determining if a glyph is in them.